package/harvester-os/files/etc/cos/bootargs.cfg

```diff
@@ -1,9 +1,9 @@
 set console_params="console=tty1"
 set kernel=/boot/vmlinuz
 if [ -n "$recoverylabel" ]; then
-    set kernelcmd="$console_params root=live:LABEL=$recoverylabel rd.live.dir=/ rd.live.squashimg=$img panic=5"
+    set kernelcmd="$console_params root=live:LABEL=$recoverylabel rd.live.dir=/ rd.live.squashimg=$img panic=5 net.ifnames=1"
 else
-    set kernelcmd="$console_params root=LABEL=$label cos-img/filename=$img panic=5"
+    set kernelcmd="$console_params root=LABEL=$label cos-img/filename=$img panic=5 net.ifnames=1"
```

Comment:
Is It the key modification to set net.ifnames as 1 to make the network interfaces predictable? Referring to https://man7.org/linux/man-pages/man8/systemd-udevd.service.8.html, it seems that the default value of net.ifnames is 1.
```
       net.ifnames=
           Network interfaces are renamed to give them predictable names
           when possible. It is enabled by default; specifying 0
           disables it.
```

Reply:
it disabled by default in leap 15.3 from my test.